config/helm/appmesh-controller/ci/values.yaml

```diff
@@ -5,5 +5,5 @@ accountId: 123456789
 region: us-west-2
 image:
   repository: public.ecr.aws/appmesh/appmesh-controller
-  tag: v1.8.0
+  tag: v1.9.0
```

Comment:
Our next controller release is 1.9.0?

Reply:
There was a v1.8.0 beta release that never got promoted. We could have tagged this 1.8.1, but opted to just do a 1.9.0 instead.

Reply:
got it